src/main/webapp/user-private-comments.jsp

```diff
@@ -51,10 +50,10 @@
             <%request.setAttribute("clickable", false);%>
             <jsp:include page="task-status-checkmark.jsp"/>
             <p class="inline"><%=task.getDescription()%></p>
-            <form onload="init(box-<%=task.getTaskID()%>, <%=(privateCommentsMap.get(task.getTaskID())).getMessage().trim());" action="/user-profile" method="POST">
+            <form action="/user-profile" method="POST">
               <input type="hidden" name="taskID" value="<%=task.getTaskID()%>"> 
               <br>
-              <div id="box-<%=task.getTaskID()%>" type="text"></div>
+              <div id="box-<%=task.getTaskID()%>" onload="init('box-<%=task.getTaskID()%>', '<%=(privateCommentsMap.get(task.getTaskID())).getMessage().trim()%>')" type="text"></div>
```

Comment:
this line has a null pointer somewhere. I'll play around with prints and readjusting quotation marks, etc...